### DISCUSSION TITLE
J4.1 Plugin installer enhancement

### DISCUSSION THREAD

**Discussion by BrainforgeUK** [joomla/joomla-cms#47115] [2. Cold Features]
### Steps to reproduce the issue

Install a plugin

### Expected result

Button providing a quick link to plugin list following installation, like the existing manage extensions button.

### Actual result

Would be nice to have a Manage Plugins button as well.

This button would pre-populate the plugin filter criteria to only show the plugin which has just been installed. Displaying in descending ID order instead would be useful when installing a new plugin but may not be so useful after an update.

**Comment by brianteeman** [comment]
Many extension developers already do this
